### PR TITLE
Add ptp labels on the nodes

### PR DESCRIPTION
**Pull request by SchSeba**
This PR adds a label of grandmaster on the first master and all the other nodes (master/workers) are labeled as slaves

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: hack/setup-test-cluster.sh [hunk 1]
```diff
@@ -13,6 +13,20 @@ do
...
+# Label First master as grandmaster
+echo "[INFO]: Labeling first master as the ptp grandmaster"
+nodes=$(${OC_TOOL} get nodes --selector='node-role.kubernetes.io/master' -o name | sed -n 1p)
+for node in $nodes
+do
+    ${OC_TOOL} label $node ptp/grandmaster=""
+done
+
+echo "[INFO]: Labeling all the other hosts as ptp slaves"
+nodes=$(${OC_TOOL} get nodes -o name | sed 1d)
```
Comment (fedepaol) [inline hack/setup-test-cluster.sh]: If you do like this, you will skip the first result which is not necessarily the master you labelled above. Is that fine?

Comment (SchSeba) [reply hack/setup-test-cluster.sh]: I make a change in the first loop can you take a look?

**Pull request merged**